tests/hash-backend-performance/src/session.ts

```diff
@@ -7,63 +7,45 @@ import { extractOwnedByIdFromEntityId } from "@local/hash-subgraph";
 import { Configuration, FrontendApi } from "@ory/client";
 
 import { getUserByKratosIdentityId } from "./graph";
-import type { BeforeRequest, RequestParams } from "./types";
+import type { ActionFn, BeforeRequestFn } from "./types";
 
 const API_ORIGIN = "http://127.0.0.1:5001";
 
 type LoginContext = {
-  flowId: string;
   session: {
     token: string;
-    expiresAt?: string;
+    expiresAt?: number;
     user: SimpleProperties<User["properties"]>;
     ownedById: OwnedById;
   };
 };
 
-const setAuthorizationHeader = (request: RequestParams, token: string) => {
-  if (!request.headers) {
-    request.headers = {};
+let __oryKratosClient: FrontendApi | undefined;
```

Comment:
Is there a naming pattern we use for this? Or is there a better way? Generally I just tried to avoid creating the client multiple times.

Reply:
I'd use one underscore (`_oryKratosClient`) but this is fine